samples/managed-app-sample/parameterFile.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
You don't need this file here. This sample is for managed app, which reference an existing k8s app